test/master-playlist-controller.test.js

```diff
@@ -2537,16 +2573,34 @@ QUnit.test('calculates dynamic BUFFER_LOW_WATER_LINE', function(assert) {
   currentTime = 10;
 
   assert.equal(mpc.bufferLowWaterLine(), 5,
-    'dynamic BLWL increases by currentTime * rate');
+    'dynamic BLWL increases by elapsedSinceStart() * rate');
 
   currentTime = 60;
 
-  assert.equal(mpc.bufferLowWaterLine(), 30, 'dynamic BLWL uses max value');
+  assert.equal(mpc.bufferLowWaterLine(), 30, 'dynamic BLWL uses max value after 60s');
 
   currentTime = 70;
 
   assert.equal(mpc.bufferLowWaterLine(), 30, 'dynamic BLWL continues to use max value');
 
+  mpc.setCurrentTime(120);
+  currentTime = 120;
+
+  assert.equal(mpc.bufferLowWaterLine(), 0, 'dynamic BLWL uses starting value at 0s after seek');
+
+  currentTime = 130;
+
+  assert.equal(mpc.bufferLowWaterLine(), 5,
+    'dynamic BLWL increases by elapsedSinceStart() * rate');
+
+  currentTime = 180;
+
+  assert.equal(mpc.bufferLowWaterLine(), 30, 'dynamic BLWL uses max value at 60s after seek');
```

Comment:
This is technically not after a seek, right?

Reply:
Same comment as above.